### DISCUSSION TITLE
How to use `Unchecked` block in Chisel

### DISCUSSION THREAD

**Discussion by Jagadesh-Ronanki** [foundry-rs/foundry#7138] [Help] [answered]
This is the issue I'm facing. Thanks in advance 

![image](https://github.com/foundry-rs/foundry/assets/106180776/d43b7e55-81b3-4174-9738-23e9f9702056)

**Comment by alex0207s** [comment, marked as answer]
You need to add a semicolon after a statement, like this: `unchecked { type(uint256).max; }`.
